crates/bevy_ecs/src/system/config/system_config.rs

```diff
@@ -1,5 +1,6 @@
 use crate::{schedule::*, system::InsertionPoint};
 
+/// Each system has one of these. It is updated using the various traits seen in this [directory](super).
```

Comment:
```suggestion
/// Controls when and how a [`System`] is run
///
/// The traits in this [module](super) are used to update these settings.
```

Not thrilled with this, but it's an improvement

Reply:
My grammar isn't amazing, sorry